### ISSUE TITLE
tweek blog to meeting good practices

### ISSUE THREAD

**Issue by JosephMcArthur** [OAButton/discussion#1675]
**Is your feature request related to a problem? Please describe.**

Our new blog fails some accessibility / best practice tests in a silly way. It also has a few other small annoying things.

Home: https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fblog.openaccessbutton.org
Post: https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fblog.openaccessbutton.org%2Fhow-we-planned-for-a-pandemic%2F

**Describe the solution you'd like**

- [x] on landing there are lots of author bylines with the same image. It's a single author blog, these aren't needed. Remove the images & names, but keep the date and reading times. Similar issues happen on posts.
- [x] subtitles / dates fail accessibility tests on homepage. 
- [x] use inter for fonts to align with style guide
- [x] Removing jquery causes the header to stop working. If it's easy to fix, lets as it would be nice to not need jquery.

Some of this will require theme level edits, but probably we should keep those limited to keep down the work required when bumping versions. Whatever we can do via the head probably makes sense to do: https://aspirethemes.com/blog/google-font-ghost-theme. 

Repo for code is here: https://github.com/OAButton/ghost-blog-theme. It has to be uploaded the the ghost backend [here](https://oabutton.ghost.io/ghost/#/settings/general).

**Comment by JosephMcArthur** [comment]
- [ ] remove double hero: https://hooshmand.net/ghost-blog-different-hero-image-feature-image/ / https://www.aventistech.com/2018/07/disable-feature-image-in-post-in-ghost-blog/
- [ ] remove double excerpt (likely can be done as above)
- [ ] add a way to 'get updates' https://hooshmand.net/modifying-ghost-3-default-theme-casper/
- [ ] add an issue to support a better 'at a glance' about section

Nice to haves:

- [ ] A way to collect sign ups of interest: https://hooshmand.net/html-contact-forms-in-ghost/
- [ ] https://github.com/gmfmi/searchinghost-easy#installation

**Comment by sssoz** [comment]
Do we consider this closed now that we’ve settled on keeping jQuery?

**Comment by JosephMcArthur** [comment]
We do!

**Comment by JosephMcArthur** [comment]
Ah, no, we should just tidy up stuff. So, let's finish the discussion on the PR & make sure we commit whatever code we end up running to the repo.

But this is now not urgent and can wait till later in the week.
